src/theory/arith/nonlinear_extension.cpp

```diff
@@ -2910,6 +2920,439 @@ std::vector<Node> NonlinearExtension::checkTranscendentalMonotonic() {
   return lemmas;
 }
 
+std::vector<Node> NonlinearExtension::checkTranscendentalTangentPlanes()
+{
+  std::vector<Node> lemmas;
+  Trace("nl-ext") << "Get tangent plane lemmas for transcendental functions..."
+                  << std::endl;
+
+  // this implements Figure 3 of "Satisfiaility Modulo Transcendental Functions
+  // via Incremental Linearization" by Cimatti et al
+  for (std::map<Kind, std::map<Node, Node> >::iterator it =
```

Comment:
You are using the namespace std and CVC4::kind in this file. Either remove them or remove the prefixes.